### PR TITLE
fix OCP-24106 OCP-25761

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei could you help to review? Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.5/common_ui_elements.xyaml [hunk 1]
```diff
@@ -178,19 +178,19 @@ wait_dropdown_loaded:
     timeout: 20
 # click and check the first item from resource list such as projects, pods, etc.
 click_first_item_from_resource_list_and_check_breadcrumb:
-  action: wait_until_no_loading
+  action: wait_box_loaded
```
Comment (SSshahan) [inline lib/rules/web/admin_console/4.5/common_ui_elements.xyaml:181]: `wait_until_no_loading` did not take effect on waiting load

File: lib/rules/web/admin_console/4.5/goto.xyaml [hunk 2]
```diff
@@ -180,6 +180,9 @@ goto_operator_subscription_page:
   url: operatorhub/subscribe?pkg=<package_name>&catalog=<catalog_name>&catalogNamespace=openshift-marketplace&targetNamespace=<target_namespace>
 goto_one_project_subscription_page:
   url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~Subscription/<subscription_name>
+goto_csv_detail_page:
+  action: wait_box_loaded
+  url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/goto.xyaml:185]: Should it be 
```
goto_csv_detail_page:
  url: k8s/ns/<project_name>/operators.coreos.com~v1alpha1~ClusterServiceVersion/<csv_name>
  action: wait_box_loaded
```

Comment (yapei) [reply lib/rules/web/admin_console/4.5/goto.xyaml:185]: Also it is also added in #918 , right?

Comment (SSshahan) [reply lib/rules/web/admin_console/4.5/goto.xyaml:185]: OK, I was plan to remove the action after giving the pass log.

**Comment by yapei** [comment]
Thanks for update

**Comment by SSshahan** [comment]
@lxia could you help to merge? Thanks

File: lib/rules/web/admin_console/4.2/common_ui_elements.xyaml [hunk 3]
```diff
@@ -142,10 +142,6 @@ click_first_item_from_sub_resource_list_and_check_breadcrumb:
   action: click_first_sub_item_in_resource_detail_list
   action: check_link_in_breadcrumb
 # click and check the first item from resources listed in grid cell, such as installed operator
```
Comment (liangxia) [inline lib/rules/web/admin_console/4.2/common_ui_elements.xyaml:144]: Is the comment related with the code ? Should we remove it as well ?

**Comment by SSshahan** [comment]
Yes. Updated

**Pull request merged**